// react-jason/themes/index.js
"use strict";
var __createBinding = (this && this.__createBinding) || (Object.create ? (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    Object.defineProperty(o, k2, { enumerable: true, get: function() { return m[k]; } });
}) : (function(o, m, k, k2) {
    if (k2 === undefined) k2 = k;
    o[k2] = m[k];
}));
exports.__esModule = true;
exports.vscodeLight = exports.vscodeDark = exports.monokaiSublime = exports.monokai = exports.github = exports.anOldHopeClassic = void 0;
var anOldHopeClassic_1 = require("./anOldHopeClassic");
__createBinding(exports, anOldHopeClassic_1, "default", "anOldHopeClassic");
var github_1 = require("./github");
__createBinding(exports, github_1, "default", "github");
var monokai_1 = require("./monokai");
__createBinding(exports, monokai_1, "default", "monokai");
var monokaiSublime_1 = require("./monokaiSublime");
__createBinding(exports, monokaiSublime_1, "default", "monokaiSublime");
var vscodeDark_1 = require("./vscodeDark");
__createBinding(exports, vscodeDark_1, "default", "vscodeDark");
var vscodeLight_1 = require("./vscodeLight");
__createBinding(exports, vscodeLight_1, "default", "vscodeLight");
